StPetersburg/InEtn3052887b.xml

```diff
@@ -0,0 +1,142 @@
+                    </origin>
+                  <acquisition>The manuscript was handed over from the <placeName ref="INS0916RussianMuseum"/> to the <placeName ref="INS0524InEtn"/>
+                  in <date>1925</date> as part of the collection of <persName ref="PRS6260Leontiev"/>.</acquisition>
+                </history>
+                <additional>
+                    <adminInfo>
+                        <recordHist>
+                            <source>
+                              <listBibl type="catalogue">
+                                <bibl>
+                                  <ptr target="bm:Chernetsov1975Svitki"/>
+                                  <citedRange unit="page">218</citedRange>
+                                </bibl>
+                                <bibl>
+                                    <ptr target="bm:Platonov1996Katalog"/>
+                                    <citedRange unit="page">70-71</citedRange>
+                                </bibl>
+                              </listBibl>
+                              <listBibl type="secondary">
+                                 <bibl>
+                                    <ptr target="bm:Platonov2017RukKniga"/>
+                                      <citedRange unit="page">199</citedRange>
+                                 </bibl>
+                              </listBibl>
+                            </source>
+                        </recordHist>
+                    </adminInfo>
+                </additional>
+              </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="NewTestament"/>
+                    <term key="Bible"/>
+                    <term key="Translation"/>
+                    <term key="DidacticMaterial"></term>
+                </keywords>
+            </textClass>
+            <langUsage>
+              <language ident="en">English</language>
+              <language ident="gez">Gǝʿǝz</language>
+              <language ident="am">Amharic</language>
+          </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DE" when="2020-05-05">Created entity, added description and content.</change>
+        </revisionDesc>
+    </teiHeader>
+    <text xml:base="https://betamasaheft.eu/">
+        <body>
+            <div type="bibliography"/>
+         <!---->
```

Comment:
a relation between a and b could be helpful, too

Reply:
I went through the list of relations but could not find an appropriate. Do you have a idea which one I could use?

Reply:
if the sdc relations are not helpful you can always use a dc:relation with a desc

Reply:
Sorry, I do not understand what you are trying to state a relation about. We can make a statement about a previous UniCirc with a hypothetic annotation, it does not need to be 100% sure (what is so, after all...)

Reply:
If you explain in words briefly, I will be glad to try and translate in appropriate relations

Reply:
Thank you Pietro, it might have been that InEtn3052887a and InEtn3052887b once constitued one single scroll. However, this is just a hypothesis

Reply:
I would add something like this
`<relation active="InEtn3052887a#UniCirc3" name="sdc:hasCertainty" passive="sdc:Hypothetique"/>
<relation active="InEtn3052887a#UniCirc3" name="sdc:undergoesTransformation" passive="InEtn3052887a#tr1"/>
<relation active="InEtn3052887a#tr1" name="sdc:hasCertainty" passive="sdc:Hypothetique"/>
<relation active="InEtn3052887a#tr1" name="sdc:hasTransformationModel" passive="sdc:D3"/>
<relation active="InEtn3052887a#tr1" name="sdc:produces" passive="InEtn3052887a"/>
<relation active="InEtn3052887a#tr1" name="sdc:produces" passive="InEtn3052887b"/>`

Reply:
@DariaElagina do you want to add the relations? then the branch can be merged

Reply:
Done. Thank you @PietroLiuzzo !